.vscode/tasks.json

```diff
@@ -0,0 +1,14 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "npm",
+			"script": "start",
+			"problemMatcher": [
+				"$eslint-stylish"
+			],
+			"label": "npm: start",
+			"detail": "react-scripts start"
+		}
+	]
+}
```

Comment:
nit: We can probably throw this in the `.gitignore` by adding the `/.vscode` directory in there I think